examples/dev_coach_review_prd.py

```
@@ -0,0 +1,846 @@
+        )
+        if not result.success:
+            print(f"Failed to create {agent_id}: {result.error}")
+            return
+        print(f"  Created: {agent_id} ({desc})")
+
+    print("\nWaiting for agents to initialize...")
+    await asyncio.sleep(5)
+
+    # Setup files
+    Path(log_file).unlink(missing_ok=True)
+    with open(output_file, "w") as f:
```

Ruff (0.14.10)

Async functions should not open files with blocking methods like `open`

(ASYNC230)